prboom2/src/r_bsp.c

```diff
@@ -434,8 +434,7 @@ static void R_AddLine (seg_t *line)
       maxdrawsegs = newmax;
     }
 
-    if(curline->miniseg == false) // figgi -- skip minisegs
-      curline->linedef->flags |= ML_MAPPED;
+    curline->linedef->flags |= ML_MAPPED;
```

Comment:
What happened to the `if`?

Reply:
It was redundant with the check in `R_Subsector` before calling `R_AddLine`